AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/AWSAPICategoryPluginTests.swift

```diff
@@ -256,4 +256,67 @@ class AWSAPICategoryPluginTests: XCTestCase {
         }
         return count
     }
+
+    /// - Given: Datastore plugin
+    /// - When: Plugin is configured, started and cleared
+    /// - Then: Plugin's publisher should not received .finished and dataStorePublisher should not be nil
+
+    func testStorageEngineClear() throws {
```

Comment:
i was originally thinking whether this test should even exist and if the additional assert should just live in `testStorageEngineStartClearStart`. I think it should because the sequence start -> clear -> start should ensure that the publisher is not finished. so this test is a subset of that one